src/common/home/sections/banner.jsx

```diff
@@ -0,0 +1,16 @@
+import { lazy, Suspense } from "react";
+import "./banner.css";
+
+const LazyBanner = lazy(() =>
```

Comment:
wouldn't this always import HackathonBanner
Remove true or replace it with a variable

Reply:
oh yes, you're right. I forgot to revert this when i was testing it. will resolve all your other comments. thanks.